parties/opkclient.go

```diff
@@ -21,55 +24,87 @@ type MFACos interface {
 }
 
 type OpkClient struct {
-	PktJson     []byte
-	Signer      *pktoken.Signer
+	Pkt         *pktoken.PKToken
```

Comment:
It seems strange to have a `pkt` as part of the OpkClient object, but the signer which uses that pkt isn't part of the struct.

Why not have both the signer and the pktoken as part of the struct or exclude both?

Reply:
I kept the pkt token included because we use it for the MFA cosigner stuff later but we never use the signer again. This might change in the future but I'm just building to what's there right now